sdk/storage/azure-storage-blobs/test/ut/blob_container_client_test.cpp

```diff
@@ -1468,4 +1468,25 @@ namespace Azure { namespace Storage { namespace Test {
         = Blobs::BlobContainerClient(m_blobContainerClient->GetUrl(), credential, clientOptions);
     EXPECT_THROW(containerClient.GetProperties(), StorageException);
   }
+
+  TEST_F(BlobContainerClientTest, ErrorCode)
+  {
+    auto inexistent = m_blobContainerClient->GetBlobClient(RandomString());
```

Comment:
Nit: rather than using a randomly generated string, hard-code to a known non-existing string for consistency.

Reply:
Random resource name is widely used in storage tests.

When we run or debug a test in LIVE mode, if the previous test crashes or is terminated in debugger, the cleanup won't be executed. So next test we run or debug the same test cases, the blob/container is usually already existing and usually leads to unwanted result. This can be avoided by using a randomly-generated blob/container name.

In addition, it works well with recording-playback framework. When running in record or playback mode, this random function always return the same name,